tron/core/actionrun.py

```diff
@@ -1046,7 +1176,7 @@ def submit_command(self, attempt: ActionRunAttempt) -> Optional[KubernetesTask]:
                 service_account_name=attempt.command_config.service_account_name,
                 ports=attempt.command_config.ports,
             )
-        except Exception:
+        except InvariantException:
```

Comment:
technically, we could have other exceptions thrown here (e.g., `build_environment()` or `filehandler.OutputStreamSerializer()` could throw - but those do seem somewhat unlikely

Reply:
hmm should we change this to ``except Exception`` to catch all other exceptions based on the hierarchy  shown here: https://martinxpn.medium.com/exception-hierarchy-python-58-100-days-of-python-9d8585e6569b#:~:text=The%20Exception%20Hierarchy,class%20in%20the%20exception%20hierarchy.

Reply:
yea, unless we're extra-sure that none of the other things we do here will throw, it's probably a good idea to change this back to `except Exception`

Reply:
or 
```
except (InvariantException, Exception):
```

Reply:
I'm honestly not sure if anything else throws or not since I didn't really touch this code, I will change it in a separate follow up PR  and not here

Reply:
LOL, I'm so confused as of why it shows  as added in this PR. This change was added three years ago: https://github.com/Yelp/Tron/blame/b1118433b10c06a6f15b668d2f5c184115d701e5/tron/core/actionrun.py

Reply:
according to this PR it should be Exception: https://github.com/Yelp/Tron/pull/926/files , not sure what brought it back in this PR